content/en/network_monitoring/devices/profiles.md

```diff
@@ -64,6 +64,29 @@ Profiles can be used interchangeably, meaning devices that share MIB dependencie
 
 For more Datadog provided profiles, see the [GitHub repository][3].
 
+### Metadata definition by profile
+
+Profiles have a metadata section that can be used to define where and how metadata should be collected. Values can be static or come from an OID value.
```

Comment:
```suggestion
In the profiles's metadata section, you can define where and how metadata is collected. Values can be static or come from an OID value.
```

Reply:
addressed in [87b8c3e](https://github.com/DataDog/documentation/pull/21347/commits/87b8c3e330917fd1b1ac4d3e80c63b47471e152f)